### ISSUE TITLE
Importing extensions via flask.ext.* is deprecated, use flask_* instead

### ISSUE THREAD

**Issue by jeffwidman** [realpython/discover-flask#3]
Details: https://github.com/mitsuhiko/flask/issues/1135

**Comment by mjhea0** [comment]
Thanks @jeffwidman. Will update when I have time.

**Comment by leslie-alldridge** [comment]
@mjhea0 are you still maintaining this repo? There's a few updates it needs such as this one and issue #14. I'm happy to help merge them over if you're stuck for time.

**Comment by mjhea0** [comment]
No, I'm not maintaining it anymore. I don't run Real Python anymore. You'll have to check with @dbader .
